lib/v2/gamegene/news.js

```diff
@@ -0,0 +1,51 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseRelativeDate, parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const url = 'https://gamegene.cn/news';
+    const { data: response } = await got({
+        method: 'get',
+        url,
+    });
+    const $ = cheerio.load(response);
+    const list = $('div.mr245')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            const aEle = item.find('a').first();
+            const href = aEle.attr('href');
+            const title = aEle.find('h3').first().text();
+            const author = item.find('a.namenode').text();
+            const category = item.find('span.r').text();
+            const dateTime = $(item.find('div.meta').find('span')[1]).text();
+            const pubDate = isNaN(Date.parse(dateTime)) ? parseRelativeDate(dateTime) : parseDate(dateTime);
```

Comment:
The date from the webpage in `href` seems always a exact date instead of relative date. You can parse the date there instead.

Reply:
![image](https://github.com/DIYgod/RSSHub/assets/11805860/c4dfb304-b280-40dc-8c22-0484e11e5879)
If there are recent posts, the time displayed will be relative time, this

Reply:
> The date from the webpage in `href`

I mean the web pages from each `link`(href) from `list` not the one from `url`.

Reply:
Sorry, I misunderstood what you meant and I have corrected it.